src/main/java/org/embulk/filter/column/ColumnVisitorImpl.java

```diff
@@ -244,13 +244,13 @@ public void jsonColumn(Column outputColumn)
                 pageBuilder.setNull(outputColumn);
             }
             else {
-                String jsonPath = new StringBuilder("$.").append(outputColumn.getName()).toString();
+                String jsonPath = new StringBuilder("$['").append(outputColumn.getName()).append("']").toString();
```

Comment:
Here is the escape method, https://github.com/jayway/JsonPath/blob/c2c1686139a128c08578cab377376ffb4b73c623/json-path/src/main/java/com/jayway/jsonpath/internal/Utils.java#L96